docs/configuring-polaris-for-production.md

```diff
@@ -0,0 +1,78 @@
+
+* **oauth2**
+  - Configure [OAuth](https://oauth.net/2/) with this setting, including a token broker
+
+* **callContextResolver** & **realmContextResolver**
+  - Use these configurations to specify a service that can resolve a realm from your bearer tokens.
+  - The service(s) used here must implement the relevant interfaces (e.g. [CallContextResolver](https://github.com/polaris-catalog/polaris/blob/8290019c10290a600e40b35ddb1e2f54bf99e120/polaris-service/src/main/java/io/polaris/service/context/CallContextResolver.java#L27)).
+
+* **authenticator.tokenBroker**
+  - Ensure that this setting reflects the token broker specified in **oauth2** above
+
+* **cors**
+  - Use settings like **cors.allowed-origins** to control what connections your Polaris service will accept
+
+
+## Metastore Management
+
+Use the configuration `metaStoreManager` to configure a [MetastoreManager](https://github.com/polaris-catalog/polaris/blob/627dc602eb15a3258dcc32babf8def34cf6de0e9/polaris-core/src/main/java/io/polaris/core/persistence/PolarisMetaStoreManager.java#L47) implementation where Polaris entities will be persisted. 
+
+The default `in-memory` implementation is meant for testing and not suitable for production usage. Instead, consider an implementation such as `eclipse-link` which allows you to store metadata in a remote database.
+
+Credentials and metadata will be stored in the metastore manager, and so be sure that your metastore manager is accordingly secured.
+
+### Configuring EclipseLink
+
+To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it.
+
+### Bootstrapping
+
+Before using Polaris when using a metastore manager other than `in-memory`, you must **bootstrap** the metastore manager. This is a manual operation that must be performed **only once** in order to prepare the metastore manager to integrate with Polaris. When the metastore manager is bootstrapped, any existing Polaris entities in the metastore manager may be **purged**.
+
+To bootstrap Polaris, run a command like the following:
+
+```bash
+java -jar /path/to/jar/snowflake-polaris-all.jar bootstrap polaris-server.yml
+```
+
+Afterwards, you can launch Polaris normally with a command like the following:
+
+```bash
+java -jar /path/to/jar/snowflake-polaris-all.jar server polaris-server.yml
+```
+
+## Other Configurations
+
+When deploying Polaris in production, consider adjusting the following configurations:
+
+* **featureConfiguration.SUPPORTED_CATALOG_STORAGE_TYPES**
+  - By default, the `FILE` storage type may be supported. This is intended for testing, and in produciton you'll likely want to disable it
```

Comment:
By default Polaris catalogs are allowed to be located in local filesystem with the `FILE` storage type. This should be disabled for production systems.